code/modules/mob/living/simple_animal/hostile/terror_spiders/green.dm

```diff
@@ -102,6 +101,5 @@
 
 /obj/structure/spider/terrorweb/green/web_special_ability(mob/living/carbon/C)
 	if(istype(C))
-		if(C.eye_blurry < 60)
-			C.AdjustEyeBlurry(30)
+		C.AdjustEyeBlurry(60 SECONDS, 0, 120 SECONDS)
```

Comment:
why the upper bound?

Reply:
`if(C.eye_blurry < 60)` to retain the original functionality